testing/e2e/src/routes/tools-test.tsx

```
@@ -1,6 +1,7 @@
 import { useState, useCallback, useRef, useEffect, useMemo } from 'react'
 import { createFileRoute } from '@tanstack/react-router'
 import { useChat, fetchServerSentEvents } from '@tanstack/ai-react'
+import type { UIMessage } from '@tanstack/ai-react'
```

ESLint

[error] `@tanstack/ai-react` type import should occur after import of `zod`

(import/order)